packages/react-aria-components/docs/FileTrigger.mdx

```diff
@@ -0,0 +1,195 @@
+  --text-color-disabled: var(--spectrum-alias-text-color-disabled);
+  --focus-ring-color: slateblue;
+
+  color: var(--text-color);
+  background: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  appearance: none;
+  vertical-align: middle;
+  font-size: 1.2rem;
+  text-align: center;
+  margin: 0;
+  outline: none;
+  padding: 4px 12px;
+
+  &[data-pressed] {
+    box-shadow: inset 0 1px 2px rgb(0 0 0 / 0.1);
+    background: var(--background-color-pressed);
+    border-color: var(--border-color-pressed);
+  }
+
+  &[data-focus-visible] {
+    border-color: var(--focus-ring-color);
+    box-shadow: 0 0 0 1px var(--focus-ring-color);
+  }
+
+  &:disabled {
+    border-color: var(--border-color-disabled);
+    color: var(--text-color-disabled);
+  }
+}
+
+@media (forced-colors: active) {
+  .react-aria-Button {
+    forced-color-adjust: none;
+    --border-color: ButtonBorder;
+    --border-color-pressed: ButtonBorder;
+    --border-color-disabled: GrayText;
+    --background-color: ButtonFace;
+    --background-color-pressed: ButtonFace;
+    --text-color: ButtonText;
+    --text-color-disabled: GrayText;
+    --focus-ring-color: Highlight;
+  }
+}
+```
+
+</details>
+
+## Features
+
+A file input can be created with an `<input type=“file”>` element, but this supports limited styling options and may not integrate well with the overall design of a website or application. To overcome this, `FileTrigger` extends the functionality of the standard file input element by working with a pressable child such as a `Button` to create accessible file inputs that can be style as needed.
+
+* **Customizable** – Works with any pressable React Aria or React Spectrum component, and custom components built with [usePress](usePress.html).
+
+## Anatomy
+
+A `FileTrigger` wraps around a pressable child such as a button, and includes a visually hidden input element that allows the user to select files from their device.
+
+If a visual label is not provided on the pressable child, then an `aria-label` or `aria-labelledby` prop must be passed to identify the file trigger to assistive technology.
+
+### Composed Components
+A `FileTrigger` can use the following components, which may also be used standalone or reused in other components.
+
+<ExampleCard
+  url="Button.html"
+  title="Button"
+  description="A button allows a user to perform an action with a mouse, touch, or keyboard.">
+  <Button />
+</ExampleCard>
+
+## Props
+
+<PropTable component={docs.exports.FileTrigger} links={docs.links} />
+
+## Styling
+
+React Aria components can be styled in many ways, including using CSS classes, inline styles, utility classes (e.g. Tailwind), CSS-in-JS (e.g. Styled Components), etc. Unlike other React Aria components, `FileTrigger` does not have a builtin `className` attribute following the `react-aria-ComponentName` naming convention. Instead, it is styled by its pressable children, such as `Button`.  
```

Comment:
Wondering if we should just remove the styling section completely for this one since styling a button is covered on the button page? Or change it to say something like we have for DialogTrigger/TooltipTrigger? https://reactspectrum.blob.core.windows.net/reactspectrum/f167e7f31c3e00033cf52520ce52f3de4d8d5ccf/docs/react-aria/Tooltip.html#tooltiptrigger-1 The first sentence here isn't really relevant, and we could just say that since it doesn't render any elements of its own it doesn't support styling.